gearoenix/CMakeLists.txt

```diff
@@ -45,7 +45,29 @@ SET(GX_ENGINE_TEST_SRC
         ${GX_ENGINE_VULKAN_TEST_SRC}
         PARENT_SCOPE)
 
+IF (MSVC)
+	SOURCE_GROUP("ai" FILES ${GX_ENGINE_AI_SRC})
+	SOURCE_GROUP("audio" FILES ${GX_ENGINE_AUDIO_SRC})
+	SOURCE_GROUP("core" FILES ${GX_ENGINE_CORE_SRC})
+	SOURCE_GROUP("dxr" FILES ${GX_ENGINE_DXR_SRC})
+	SOURCE_GROUP("math" FILES ${GX_ENGINE_MATH_SRC})
+	SOURCE_GROUP("physics" FILES ${GX_ENGINE_PHYSICS_SRC})
+	SOURCE_GROUP("platform" FILES ${GX_ENGINE_PLATFORM_SRC})
+	SOURCE_GROUP("render" FILES ${GX_ENGINE_RENDER_SRC})
+	SOURCE_GROUP("vulkan" FILES ${GX_ENGINE_VULKAN_SRC})
+	SOURCE_GROUP("imgui" FILES "${GX_IMGUI_PATH}/imgui.cpp")
+	SOURCE_GROUP("imgui" FILES "${GX_IMGUI_PATH}/imgui_tables.cpp")
+	SOURCE_GROUP("imgui" FILES "${GX_IMGUI_PATH}/imgui_widgets.cpp")
+	SOURCE_GROUP("imgui" FILES "${GX_IMGUI_PATH}/imgui_draw.cpp")
+	SOURCE_GROUP("imgui" FILES "${GX_IMGUI_PATH}/imgui_demo.cpp")
+	SOURCE_GROUP("imgui" FILES "${GX_IMGUI_PATH}/imgui_widgets.cpp")
+	SOURCE_GROUP("imgui" FILES "${GX_IMGUI_PATH}/backends/imgui_impl_vulkan.cpp")
+	SOURCE_GROUP("imgui" FILES "${GX_IMGUI_PATH}/misc/cpp/imgui_stdlib.cpp")
+	SOURCE_GROUP("imgui" FILES "${GX_IMGUI_FB_PATH}/ImGuiFileDialog.cpp")
+ENDIF()
```

Comment:
What is the reason that it is only for MSVC?
If there is any particular reason, maybe it is better to be commented before the `IF(MSVC)`.